mod/aalborg_theme/views/default/elements/buttons.css

```diff
@@ -18,6 +18,12 @@
 .elgg-button:focus {
 	text-decoration: none;
 }
+.elgg-button .elgg-icon {
+    color: inherit;
```

Comment:
I just really hate that I have to do this all the time. Do we really need to set color on the icons?